configs/open-access.nrel-op.json

```
@@ -7,6 +7,8 @@
     },
     "intro": {
         "program_or_study": "study",
+        "start_month": "9",
+        "start_year": "2020",
```

```suggestion
        "start_month": "8",
        "start_year": "2022",
```

The first trip is actually July 25, but maybe we should start with Aug since that is already the end of July